[2025-03-13T12:45:02.797+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-03-13T12:40:00+00:00 [queued]>
[2025-03-13T12:45:02.805+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-03-13T12:40:00+00:00 [queued]>
[2025-03-13T12:45:02.806+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-13T12:45:02.815+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_weather_task> on 2025-03-13 12:40:00+00:00
[2025-03-13T12:45:02.821+0000] {standard_task_runner.py:60} INFO - Started process 254 to run task
[2025-03-13T12:45:02.823+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'load_weather_task', 'scheduled__2025-03-13T12:40:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/etl_repeat_weather.py', '--cfg-path', '/tmp/tmpsld2fown']
[2025-03-13T12:45:02.825+0000] {standard_task_runner.py:88} INFO - Job 44: Subtask load_weather_task
[2025-03-13T12:45:02.871+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.load_weather_task scheduled__2025-03-13T12:40:00+00:00 [running]> on host 7de7158dc909
[2025-03-13T12:45:03.077+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='load_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T12:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T12:40:00+00:00'
[2025-03-13T12:45:03.185+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil disimpan. 5 rekaman baru, 10 total dalam sejarah
[2025-03-13T12:45:03.203+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=load_weather_task, execution_date=20250313T124000, start_date=20250313T124502, end_date=20250313T124503
[2025-03-13T12:45:03.238+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-13T12:45:03.251+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
